common/views/components/IframeContainer/IframeContainer.tsx

```diff
@@ -0,0 +1,72 @@
+import styled from 'styled-components';
+const IframeContainer = styled.div`
```

Comment:
Just to reduce the size of the diff, could we pull these Iframe changes into a separate PR?